README.md

```diff
@@ -1,20 +1,28 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+A click handler is an event handler that fires when the listener hear a 'click'.
+
+(b) Where in your JS file should you put it, and WHY?
+
+event handlers need to all be referenced in the scope of $(document).ready function, so that the listener has a dom to listen to.
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+
```

Comment:
Looks like you didn't get to Q2. `$ is undefined` means that you didn't require JQuery correctly.